apps/docs/pages/guides/platform/ipv4-address.mdx

```diff
@@ -0,0 +1,45 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+import { Admonition } from 'ui'
+
+export const meta = {
+  title: 'IPv4 Address',
+  description: 'Attach an IPv4 address to your database',
+  subtitle: 'Attach an IPv4 address to your database',
+}
+
+From 26th January 2024, we will begin migrating projects to IPv6. To read more about why we are doing this migration, read [this blog post](https://supabase.com/blog/ipv6) and [this discussion](https://github.com/orgs/supabase/discussions/17817) for how to prepare for the migration. As a summary, AWS will [start charging for IPv4 addresses](https://aws.amazon.com/blogs/aws/new-aws-public-ipv4-address-charge-public-ip-insights/).
+
+The add-on is $4 per month - we're passing on the AWS costs to you. The add-on is available for all paid plans.
+
+## When is this add-on required?
+
+This add-on is not required if any of these conditions are met
+
+- you are connecting to the database via Supavisor, our connection pooler.
+- you are connecting via any of our client libraries.
+- If you are connecting to the database directly and the network from which you are connecting to the database is able to resolve IPv6 addresses.
```

Comment:
the network ... supports IPv6 connections (since you can resolve and still fail to connect)